tests/unit/test-util/test.adb.js

```diff
@@ -1322,8 +1322,7 @@ describe('utils/adb', () => {
       assert.deepEqual(devices, ['emulator1', 'device2']);
     });
 
-    it('test listADBFirefoxAPKs', async () => {
-      const adb = getFakeADBKit({});
+    it('lists installed firefox apks on a device', async () => {
```

Comment:
Nit (mainly because this PR would be updated for the changes to the right, and while we are still here let's also do some nits), `it('should export a listADBFirefoxAPKs method', ...)`.